core/src/main/java/com/linecorp/armeria/server/docs/MethodInfo.java

```diff
@@ -108,6 +114,8 @@ public MethodInfo(String name,
         this.exampleHttpHeaders = ImmutableList.copyOf(requireNonNull(exampleHttpHeaders,
                                                                       "exampleHttpHeaders"));
         this.exampleRequests = ImmutableList.copyOf(requireNonNull(exampleRequests, "exampleRequests"));
+        this.examplePaths = ImmutableList.copyOf(requireNonNull(examplePaths, "examplePaths"));
+        this.exampleQueries = ImmutableList.copyOf(requireNonNull(exampleQueries, "exampleQueries"));
```

Comment:
How about validating all paths and query strings, using `PathAndQuery` and `QueryParams.fromQueryString()`?

Reply:
This is not addressed yet.